src/main/resources/application.properties

```diff
@@ -6,6 +6,8 @@ management.metrics.export.prometheus.enabled=true
 management.endpoints.web.base-path=/
 management.endpoints.web.path-mapping.prometheus=metrics
 management.endpoints.web.path-mapping.metrics=act-metrics
+management.endpoint.health.show-details=always
```

Comment:
Should this be endpoints?

Reply:
Nope, `endpoints` is a namespace for settings applicable to all endpoints, whereas `endpoint` is to configure each endpoint individually. There are some other `endpoint` settings in the file